deploy/charts/alluxio/templates/fuse/daemonset.yaml

```diff
@@ -133,7 +133,11 @@ spec:
           command: ["/bin/sh", "-c"]
           args:
             - umount -l {{ $alluxioFuseMountPoint }};
+              {{- if ne (get .Values.properties "alluxio.mount.table.source") "ETCD" }}
```

Comment:
the mount table can be specified in a static file and we share the file in `ConfigMap` or `Secret` among pods, so the mount table does not only support ETCD mode. See https://github.com/TachyonNexus/enterprise/blob/main/docsEE/en/operation/Alluxio-Namespace-and-Under-File-System-Namespaces.md for the 3 modes for mount table we are supporting: `ETCD`, `STATIC_FILE` and `NONE`

I figure the easiest way out is to check:
1. If `alluxio.mount.table.source in [ETCD, STATIC_FILE]` then mount with dataset path
2. If  `alluxio.mount.table.source = NONE` then just mount without that path and rely on `alluxio.dora.client.ufs.root`

Reply:
Do we always want user to set `alluxio.mount.table.source`? I'm thinking only if mount table source is etcd and static file, then mount without the `alluxio.dora.client.ufs.root`; if not set or set to any other value we use `alluxio.dora.client.ufs.root`

Reply:
Yea good point... The Alluxio property `alluxio.mount.table.source` defaults to `NONE` so if the user wants `ETCD` or `STATIC_FILE`, they must set it. But honestly this is subject to changes in the future. I'm honestly not sure how the operator is supposed to work independent of Alluxio version (however Alluxio changes, the operator does not rely on default). But I feel the operator better sets this value explicitly. Does that make sense?

Reply:
I see. You mean maybe in the future `NONE` might not be the default anymore.

Reply:
But it doesn't make sense to enforce OS user to set this value lol

Reply:
What do you think? As this is not an OS feature, hiding it inside Alluxio properties instead of exposing to users I think makes more sense. And we probably gonna revert it once we have the right repo